README.md

```diff
@@ -236,3 +236,5 @@ https://github.com/niilo/clamav-rest
 
 This will run a clamav-rest server at http://localhost:9000
 
+Set RESTY_SERVICE_URL to "http://localhost:9000/scan"
+
```

Comment:
Maybe mention that these new env vars need to be set in production ?   

RESTY_USERNAME and RESTY_PASSWORD and PROXY

Reply:
At the moment they don't, but will add a note